bin/heal

```diff
@@ -71,12 +72,27 @@ def main():
         action='store_true'
         )
 
+    parser.add_argument(
+        '--process-translations',
+        dest='process_translations',
+        help='Retrieves completed 3PlayMedia translations for the videos.',
+        action='store_true'
+    )
+
     args = parser.parse_args()
 
     veda_id = args.veda_id
     course_id = args.course_id
     schedule = args.schedule
 
+    process_translations = args.process_translations
+    if process_translations:
+        # Only kick off a round of retrieving successful
+        # translations from 3Play Media
+        os.environ['DJANGO_SETTINGS_MODULE'] = 'VEDA.settings.production'
```

Comment:
Do we need to add this here ?

Reply:
yes. it is to tell Django about the correct settings.

Reply:
This is temporary and will be removed once we have Django mgmt. command for this.